workspaces/kurtosis/pectra-stable.yml

```diff
@@ -2,21 +2,25 @@ participants_matrix:
   el:
     - el_type: geth
       el_image: ethereum/client-go:v1.15.7
-      el_extra_params: [--gcmode=archive]
+      el_extra_params: [ --gcmode=archive ]
+    - el_type: erigon
+      el_image: erigontech/erigon:v3.0.2
   cl:
     - cl_type: teku
-      cl_image: consensys/teku:25.4.0
+      cl_image: consensys/teku:25.4.1
       use_separate_vc: true
+      vc_extra_env_vars: { "JAVA_TOOL_OPTIONS": "-XX:UseSVE=0" }
+      cl_extra_env_vars: { "JAVA_TOOL_OPTIONS": "-XX:UseSVE=0" }
```

Comment:
As far as I remember this option will break launching on all environments except M4